### PR TITLE
Docker db

### PR DESCRIPTION
**Pull request by aszatkowska**

### CODE REVIEW

File: dev.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM node:18-alpine3.15
```
Comment (Miszczu2137) [inline dev.Dockerfile:1]: Please change name file into Dockerfile.local

File: prod.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+FROM node:18-alpine3.15 AS appbuild
```
Comment (Miszczu2137) [inline prod.Dockerfile:1]: Change filename into Dockerfile

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+version: '3.8'
```
Comment (Miszczu2137) [inline docker-compose.yml:1]: Docker-compose is only for development purpose. Prod container shouldn't be here.

File: src/config/config.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { config } from 'dotenv';
```
Comment (Miszczu2137) [inline src/config/config.ts:1]: This is your primary config ? How do you want to handle much more complex/nested structure of configuration ?

File: src/infrastructure/database.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { Client } from 'pg';
+const connectionstring = process.env.CONNECTION_STRING;
```
Comment (Miszczu2137) [inline src/infrastructure/database.ts:2]: This should comes from node-config.

File: docker-compose.yml (hunk 3 above)
Comment (aszatkowska) [reply docker-compose.yml:1]: Prod containers have been removed. All commands to run prod containers are in README.md  file

**Pull request merged**